### PR TITLE
Project 5-Update-ExpenseTracker-Angular

### PR DESCRIPTION
**Pull request by Harshi7016**

### CODE REVIEW

File: content/participants/ttjc@Harshi7016/index.md [hunk 1]
```diff
@@ -104,4 +124,10 @@ The blog was mainly focussed on a few chrome extensions which I like the most an
 
 **Visit**--[Chrome Extensions for Front-end Devs](https://medium.com/@harshithece7016/chrome-extensions-for-front-end-devs-63f8a1b7a8a9)
 
-No new blogs written as part of the job challenge. Check my [Blog](https://harshith-venkatesh-blog.netlify.app/) for previous blogs.
+#### \*Why So Much React for STATE\*\*
```
Comment (gkanishk) [inline content/participants/ttjc@Harshi7016/index.md:127]: please rectify its markdown.

**Comment by Harshi7016** [comment]
Hi Kanish I have made the changes to the markdown thanks for the update, I will update the title in the same way (y)

**Comment by Harshi7016** [comment]
*👍

**Comment by Harshi7016** [comment]
Updated all the title with the appropriate names, thanks for the update, I would have taken time to notice that.
Good Feedback

**Comment by gkanishk** [comment]
Awesome!!!
Now you can raise **PR** for the finalist.

**Pull request merged**